front-end/src/services/utils.js

```diff
@@ -0,0 +1,5 @@
+export const utilFunc = {
+    stringifyUrl: (type, value) => {
```

Comment:
Could be simplified:
```suggestion
    stringifyUrl: (type, value) =>`&${type}=${value}`
```